components/elm/cime_config/testdefs/testmods_dirs/elm/gis20kmERP/user_nl_mali

```diff
@@ -0,0 +1,8 @@
+config_velocity_solver = 'FO'
+config_thickness_advection = 'fo'
+config_thermal_solver = 'none'
```

Comment:
Do we want to be explicitly setting these settings (and the calving ones) in a test, as opposed to using the nl defaults?  Hardcoding specific physics options seems fragile because it wouldn't catch the effects of updating nl defaults.

Reply:
I think the tests need to run "out of the box" by default w/o anyone making any changes. I.e., I think they need to run just by executing the lines noted above (in the description for how to "use" capabilities in this PR under "Low-resolution tests"). Jon should be able to clarify how this should work. But note that when you create a test instance, it gets created, built, and submitted all at the same time. So there's not an obvious / easy way to break between those steps to make changes to things like namelists, xml files, etc.

Reply:
What I'm wondering is if we should choose to set these settings as the default namelist values for MALI if it's what we want running in the tests.

Reply:
I think for 1 and 2 above these should be the defaults. I don't know about 3. I could try the test w/ that set to 'temperature' as I'm guessing we may not want 'none' as the default. 

The other ones that I set in here are the following:

```
config_pio_num_iotasks = 47
config_pio_stride = 64
config_calving = 'none'
config_restore_calving_front = .true.
```

Based on the discussion w/ Jon, I can remove the PIO settings (?). But for calving, like thermal solver, I suspect we don't want these settings to be 'none' and .true. I could also re-try these tests w/ these settings not specified (so using whatever the defaults are). 







config_dt = '0000-00-01_00:00:00'

Reply:
FYI, here are the current defaults for all of the params I update as part of these namelist changes:

```
config_velocity_solver = 'FO'    [when 'mali_use_albany' = .true.]
config_thickness_advection = 'fo'
config_thermal_solver = 'none'
config_pio_num_iotasks = 0
config_pio_stride = 1
config_calving = 'none'
config_restore_calving_front = .true.
config_dt = '0000-00-01_00:00:00'
```
So I think my tests should all work fine w/o any of the changes being discussed above. I will test that and confirm, in which case I can just remove these files.

Reply:
They work w/ the defaults. I've removed these files and will push that change (along w/ one other) in a few mins. I'm going to resolve this portion of the discussion.